### ISSUE TITLE
Is this really CS 202?

### ISSUE THREAD

**Issue by jamesEmerson112** [Oniel/CS202_Complex-Number-Image-Maker#1]
This is really CS 202. To be frank, I really need to catch up this level. Did you survive this assignment?

**Comment by Oniel** [comment]
Yes this was one of the CS 202 assignments at UNLV, from several years ago (4 if I recall correctly). Yup, I survived. I didn't find the assignment to be that difficult. The assignment itself is not difficult James, the difficulty comes with ourselves we get anxious and become tunnel visioned (at least that was the case for me) and that's where the difficulty lies. But you'll get better with time, we all do. Go for a run, drink some chamomile tea, chill, and once your relaxed grab a pen and paper and go through the problem calmly. 
Hope that helped & happy coding!

**Comment by jamesEmerson112** [comment]
Thank you.
